packages/marginfi-client-v2/src/models/account/wrapper.ts

```diff
@@ -640,8 +622,8 @@ class MarginfiAccountWrapper {
     swapIxs: TransactionInstruction[],
     swapLookupTables: AddressLookupTableAccount[],
     priorityFeeUi?: number,
-    createAtas?: boolean
-    // isTxnSplitParam?: boolean
+    createAtas?: boolean,
+    broadcastType: TransactionBroadcastType = "BUNDLE"
```

Comment:
Why is there a default value here? Is it optional to pass in?

Reply:
the main reason is, other people are using this sdk and if we make a change to the required params, it will break for them. So we always gotta be very carefull with changing things around in the sdk